src/OSPSuite.TeXReporting/TeX/PGFPlots/Plot.cs

```diff
@@ -198,19 +201,20 @@ private void plotShadedErrorBars(StringBuilder text)
          text.AppendFormat(
             "\\addplot [{0}, draw=none, area legend, fill={0}, opacity={1}{2}] \n",
             Options.Color, Options.Opacity, Options.ShowInLegend ? string.Empty : ", forget plot");
-         var areaCoordinates =
```

Comment:
What a weird way to concatenate list. Not sure why the same entry (the first) would be added twice .But anyways, now that won't crash

Reply:
Overall. This LaTex code generation is terrible at best